### PR TITLE
[samples] Fix TrafficLight.js logic

### PR DESCRIPTION
**Pull request by grgustaf**
This was the worst traffic light controller in the world, warning you
with a yellow when the light was about to turn green, instead of red.

Note: Yellow is still represented by a green LED. ;)

Signed-off-by: Geoff Gustafson <geoff@linux.intel.com>

### CODE REVIEW

**Comment by brianjjones** [comment]
+1

File: samples/TrafficLight.js [hunk 1]
```diff
@@ -41,16 +41,16 @@ green.write(false);
...
+        // switch to green after 5s
         red.write(false);
-        yellow.write(true);
-    } else if (elapsed == 7) {
-        // switch to green after 2s
-        yellow.write(false);
         green.write(true);
-    } else if (elapsed == 10) {
-        // switch to red after 3s
+    } else if (elapsed == 8) {
```
Comment (jimmy-huang) [inline samples/TrafficLight.js]: Do we have to use === for the sake of good practice? not that elapsed will ever turn into another type in this case

Comment (grgustaf) [inline samples/TrafficLight.js]: Refreshing myself on that, I guess yeah that is the better practice. Let's not do that here, but feel free to submit a separate PR fixing them across all samples and tests.

**Comment by jimmy-huang** [comment]
+1

**Pull request merged**